### PR TITLE
Organizations API Updates

### PR DESCRIPTION
**Pull request by HumanistSerif**
Edited the following topics to remove incorrect syntax and/or parameters.
* Assign Account to Organization
* Replace Existing Organization Account Assignment
* Delete Existing Organization Account Assignment

### CODE REVIEW

File: _organization/org-account-assignment-add-v2.md [hunk 1]
```diff
@@ -2,12 +2,9 @@
 title: Assign Account to Organization
 position: 8
 description: Assign accounts to an organization. If your organization is in FlexOrgs, run the [Get All Allowed Accounts](#organization_get-all-allowed-accounts) endpoint first to verify which accounts you can assign to the organization.
-type: put
+type: patch
 endpoint: https://chapi.cloudhealthtech.com/v2/organizations/:org_id
 parameters:
-  - name: accounts
-    required: yes
-    content: Enter `add` to add new accounts to the organization.
```
Comment (dturcotte) [inline _organization/org-account-assignment-add-v2.md:10]: this one's a bit mixed now, there are two different operations:

 * `PUT` on `https://chapi.cloudhealthtech.com/v2/organizations/:org_id` with no `accounts` parameter: replaces existing account assignment with what's provided in the body

 * `PATCH` on `https://chapi.cloudhealthtech.com/v2/organizations/:org_id/accounts` (different endpoint, this one has `/accounts` on the end) with `"accounts": "add"`: adds to the existing account assignments, similar to how removing accounts works below

Comment (HumanistSerif) [reply _organization/org-account-assignment-add-v2.md:10]: Mind is sufficiently blow. Thanks for the clarification. I propose that we call these operations out separately, then (at the risk of repeating the endpoints).
* Assign Account to Organization
* Add Accounts to Existing Organization Account Assignment

Comment (dturcotte) [reply _organization/org-account-assignment-add-v2.md:10]: perfect - it has understandably been a sticking point with customers so I think this is a good plan

Comment (HumanistSerif) [reply _organization/org-account-assignment-add-v2.md]: @dturcotte Please take a look at this change and advise.

File: _organization/org-account-assignment-add-v2.md [hunk 2]
```diff
@@ -1,13 +1,10 @@
 ---
 title: Assign Account to Organization
 position: 8
-description: Assign accounts to an organization. If your organization is in FlexOrgs, run the [Get All Allowed Accounts](#organization_get-all-allowed-accounts) endpoint first to verify which accounts you can assign to the organization.
+description: Assign accounts to an organization. If your organization is in FlexOrgs, run the [Get All Allowed Accounts](#organization_get-all-allowed-accounts) endpoint first to verify which accounts you can assign to the organization. Existing account assignments in the organization will be entirely replaced by the assignments you specify in the body of this request. If you wnat to add to existing account assignments, see [Add Accounts to Existing Organization Account Assignment](#organization_add-accounts-to-existing-organization-account-assignment).
```
Comment (dturcotte) [inline _organization/org-account-assignment-add-v2.md:4]: just a typo, `wnat`

```suggestion
description: Assign accounts to an organization. If your organization is in FlexOrgs, run the [Get All Allowed Accounts](#organization_get-all-allowed-accounts) endpoint first to verify which accounts you can assign to the organization. Existing account assignments in the organization will be entirely replaced by the assignments you specify in the body of this request. If you want to add to existing account assignments, see [Add Accounts to Existing Organization Account Assignment](#organization_add-accounts-to-existing-organization-account-assignment).
```

File: _organization/org-account-assignment-append.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: Add Accounts to Existing Organization Account Assignment
+position: 8
+description: Add one of more accounts to an organization.
+type: patch
+endpoint: https://chapi.cloudhealthtech.com/v2/organizations/:org_id/accounts
+parameters:
+  - name: aws_accounts
```
Comment (dturcotte) [inline _organization/org-account-assignment-append.md:8]: the rest of this looks good, but I don't see the `accounts` parameter documented here (needs to be `"add"`), though I do see it in the example requests

```suggestion
  - name: accounts
    required: yes
    content: Enter `add` to assign accounts to the organization.
  - name: aws_accounts
```

**Review by dturcotte — COMMENTED** [review]
looking good! just a couple small things

File: _organization/org-account-assignment-append.md (hunk 3 above)
Comment (HumanistSerif) [reply _organization/org-account-assignment-append.md:11]: Ugh! So sorry. Too many tabs open in my editor. Fixed it now.

File: _organization/org-account-assignment-add-v2.md (hunk 2 above)
Comment (HumanistSerif) [reply _organization/org-account-assignment-add-v2.md]: Great catch!

**Review by dturcotte — APPROVED** [review]
awesome, thank you!

**Pull request merged**